app/assets/stylesheets/cortex-plugins-core/components/thumbnail-placeholder.scss

```diff
@@ -0,0 +1,12 @@
+.thumbnail-placeholder {
+  background-color: #E4E4E4; // TODO: determine how to inherit variables from Cortex. This should be $color-grey-evenlighter
```

Comment:
@arelia That'd do it! It was late that night.. We should get started on that ASAP for V3.